### PR TITLE
202204 tp107257 remove desc from atc

### PR DESCRIPTION
**Pull request by cykolln**

### CODE REVIEW

**Comment by klaviyojad** [comment]
looks good, seems confusing because the last changes were not in this branch but had to be done

**Review by siddwarkhedkar — APPROVED** [review]

**Comment by cykolln** [comment]
> looks good, seems confusing because the last changes were not in this branch but had to be done

Agreed - unfortunately there are some inconsistencies in the current branches. I'm going to be opening another feature branch after this is merged into develop which will include the changelog updates currently missing

**Comment by klaviyojad** [comment]
I agree that changelog is lacking for my update as well as Austin's

**Comment by cykolln** [comment]
> I agree that changelog is lacking for my update as well as Austin's

For simplicity I'll just do it now

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### [Unreleased]
+- Moved webhooks url to async tier
+- Add install restriction for users on < 2.4.x (must use extension v3.0.11)
+- Removed descriptions for products in Added to Cart payloads
 
 ### [4.0.2] - 2022-03-15
```
Comment (cykolln) [inline CHANGELOG.md:12]: Removed references to 4.0.3 for simplicity - potentially have one more PR going into this next release.

Comment (cykolln) [reply CHANGELOG.md:12]: All Unreleased PRs changes should now be documented correctly

**Pull request merged**